### PR TITLE
Don't check compiled assets into git

### PR DESCRIPTION
**Pull request by graysonwright**
If the files are present in git, then Heroku will try to use them
instead of compiling the assets itself. There are more details here:
https://devcenter.heroku.com/articles/rails-asset-pipeline

But in the end, it's simpler to just not check the files into git, and
let Heroku re-compile the assets when it needs to (whenever you push to
Heroku).

### CODE REVIEW

**Comment by graysonwright** [comment]
It looks like this is working now! http://dischords.herokuapp.com

**Comment by graysonwright** [comment]
Also, you should go through my pull requests and take a look at my changes. If you see anything you have a question about, ask. If you see anything I can improve, point it out and I'll fix it!

**Comment by graysonwright** [comment]
For this pull request, you should check out https://github.com/graysonwright/DFD/commit/d40950841dff8d807b78bcf2dbbbdb857c133fa8

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
 @import 'login';
 @import 'events';
 @import 'history';
+@import "parallax";
```
Comment (wrightmhw) [inline app/assets/stylesheets/application.css.scss]: I have to undo this, and add it back later. I had the correct parallax CSS elsewhere. Ill put it in a seperate file later, but this breaks the current parallax window.

**Comment by graysonwright** [comment]
How about we just remove that parallax CSS that I added? The problem was that it was in the `app/assets/javascripts` directory. Heroku couldn't compile the assets because it thought it was reading javascript when it was actually reading SCSS.

**Comment by graysonwright** [comment]
Hey, Michael! I think I fixed it. http://dischords.herokuapp.com/members

It's useful to read about [how Rails handles assets in production].
The "fingerprinting" code that Rails adds to each file is to make sure
that when the file changes, Rails serves up the newest version of the
file. The fingerprint that gets added to the filename changes whenever
the file changes, so Rails can keep track of which version of the file
is the most recent just by storing the fingerprint every time it
compiles the files.

[how Rails handles assets in production]: http://guides.rubyonrails.org/asset_pipeline.html#in-production

**Pull request merged**